server/svix-server/src/v1/utils.rs

```diff
@@ -63,6 +63,62 @@ pub trait ModelOut {
     }
 }
 
+/// Recursively searches a [`validator::ValidationErrors`] tree into a linear list of errors to be
+/// sent to the user
+fn validation_errors(
+    acc_path: Vec<String>,
+    err: validator::ValidationErrors,
+) -> Vec<ValidationErrorItem> {
+    err.into_errors()
+        .into_iter()
+        .map(|(k, v)| {
+            // Add the next field to the location
+            let mut loc = acc_path.clone();
+            loc.push(k.to_owned());
+
+            match v {
+                // If it's a [`validator::ValidationErrorsKind::Field`], then it will be a vector of
+                // errors to map to [`ValidationErrorItem`]s and insert to [`out`] before the next
+                // iteration
+                validator::ValidationErrorsKind::Field(vec) => vec
+                    .into_iter()
+                    .map(|err| {
+                        Some(ValidationErrorItem {
+                            loc: loc.clone(),
+                            // TODO: Don't skip errors without messages if possible
+                            msg: err
+                                .message
+                                .unwrap_or(Cow::Borrowed("Validation error"))
```

Comment:
```suggestion
                                .unwrap_or_else(|| Cow::Borrowed("Validation error"))
```
I'm surprised clippy didn't complain.

Reply:
In warns on your suggested change for https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_lazy_evaluations

Reply:
Yeah, I'm silly, sorry.